packages/dashboard-frontend/src/services/registry/devfiles.ts

```diff
@@ -86,16 +86,24 @@ export function updateObjectLinks(object: any, baseUrl): any {
   return object;
 }
 
+export function getRegistryIndexUrl(registryUrl: string, isExternal: boolean): URL {
+  registryUrl = registryUrl[registryUrl.length - 1] === '/' ? registryUrl : registryUrl + '/';
```

Comment:
Same as line 103

https://github.com/eclipse-che/che-dashboard/blob/fix_external_devfile_registries/packages/dashboard-frontend/src/services/registry/devfiles.ts#L103

```suggestion
```

Reply:
Thanks. I have fixed it.